README.md

```diff
@@ -282,7 +187,3 @@ I would like to thank...
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- You have done well by including this path but the path is currently broken - not working 🚯 
- [ ] Please check the link and see how you can make this work. Broken links in our works makes our work unprofessional and suggest that we don't have eyes for details.